app/search/template.hbs

```diff
@@ -115,22 +234,81 @@ as |layout|>
-                                    data-analytics-name='Object type filter={{resourceType.value}}'
-                                    data-test-topbar-object-type-link='{{resourceType.value}}'
-                                    local-class='object-type-filter-link'
-                                    @route='search'
-                                    @query={{hash resourceType=resourceType.value}}
-                                    {{on 'click' (fn this.updateResourceType resourceType)}}
-                                >
-                                    {{resourceType.display}}
-                                </LinkTo>
+                                {{#if (eq index 0)}}
+                                    <LinkTo
+                                        data-analytics-name='Object type filter={{resourceType.value}}'
+                                        data-test-topbar-object-type-link='{{resourceType.value}}'
+                                        local-class='object-type-filter-link'
+                                        @route='search'
+                                        @query={{hash resourceType=resourceType.value}}
+                                        {{on 'click' (fn this.updateResourceType resourceType)}}
+                                        id='allItems'
+                                    >
+                                        {{resourceType.display}}
+                                    </LinkTo>
+                                {{else}}
+                                    <LinkTo
+                                        data-analytics-name='Object type filter={{resourceType.value}}'
+                                        data-test-topbar-object-type-link='{{resourceType.value}}'
+                                        local-class='object-type-filter-link'
+                                        @route='search'
+                                        @query={{hash resourceType=resourceType.value}}
+                                        {{on 'click' (fn this.updateResourceType resourceType)}}
+                                    >
+                                        {{resourceType.display}}
+                                    </LinkTo>
+                                {{/if}}
                             </li>
                         {{/each}}
                     </ul>
                 </nav>
+                {{!-- template-lint-disable no-heading-inside-button --}}
+                <EmberPopover
+                    data-test-search-help-1
+                    @innerClass='tooltip-wrapper'
+                    @targetId='allItems'
+                    @side='bottom'
+                    @isShown={{this.showTooltip1}}
+                    @event='none'
+                    @spacing={{20}}
+                    local-class='search-help-tutorial'
+                    id='popover1'
+                >
+                    <aside local-class='help-content'>
```

Comment:
Do we need the `<aside>` element here? I am not sure how helpful this semantic tag is because `<EmberPopover>` already wraps a `<div>` around all the content within the block. According to [the HTML living standard](https://html.spec.whatwg.org/multipage/sections.html#the-aside-element),

> The [aside](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-aside-element.html#the-aside-element) element [represents](http://www.w3.org/TR/2011/WD-html5-20110525/rendering.html#represents) a section of a page that consists of content that is tangentially related to the content **around** the [aside](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-aside-element.html#the-aside-element) element

And if I understand correctly, an `<aside>` within a `<div>` means it contains contents "aside" to its siblings within this `<div>`, which is not what we want semantically here.

Reply:
It was between aside and a section. Regardless, the aside refers to the content that is aside from the main page, not its sibling divs. If you want to make it less semantic, I can replace it with a div.

Reply:
I was referring to the HTML living standard and its examples, specifically the third example. The wording in the standard as well as the example suggests that the `<aside>` element is for information tangentially related to the content around it. That means, depending on where the `<aside>` element is rendered in the DOM, the content "around" the aside element is different. That means it is not always "aside" to the contents in the main page, whatever the main page might be. I am afraid we are being too semantic here, and run the risk of being semantically incorrect.